iocage/lib/ioc_create.py

```diff
@@ -86,6 +86,106 @@ def create_jail(self):
             iocage.lib.ioc_destroy.IOCDestroy().destroy_jail(location)
             sys.exit(1)
 
+    def _get_template_config_fn(self, jail_uuid, path):
+        config_fn = iocage.lib.ioc_json.IOCJson(path).json_get_value
+        try:
+            config_fn("cloned_release")
+        except (IOError, OSError, FileNotFoundError, UnboundLocalError):
+            # Unintuitevly a missing template will throw a
+            # UnboundLocalError as the missing file will kick the
+            # migration routine for zfs props. We don't need that :)
```

Comment:
in general, i would ***not*** expect a function called `json_get_value()` to start a zfs migration
this should probably be fixed at the source, rather than having to wrap it like this.